### ISSUE TITLE
Deploy the "Simulation Videos"

### ISSUE THREAD

**Issue by HadadIdan** [TechnionYP5779/SimuMole#102]
Given 6 videos (assume .avi ):
* Display a web page with 7 panels: 6 small panels, each containing a video (all of the videos runs at the same time)
* Clicking on a video-panel from the six specified above opens the chosen video in the 7th panels (which is larger and covers most of the screen).
* Notice that at all points in time, the videos should all run at the same time, meaning they don't lose synchronization even if you click on one of them.
basically:
![image](https://user-images.githubusercontent.com/44426929/57644147-28cbde80-75c4-11e9-9ddd-6a798ffdcfe7.png)

(This issue is created in respond to the discussion in #92)

**Comment by HadadIdan** [comment]
We went for a different design:
![image](https://user-images.githubusercontent.com/44426929/59376803-8c7b3000-8d59-11e9-946e-244b45b10a23.png)
Showing the MD from 10 different angles. 
Beside the main (big) video, each side (except the left one) holds a rotation in a certain axsis. Up it show rotation in the Y axis, Down it shows rotation in the Z axis, and Right it shows rotation in the X axis. Left it shows the "basic" stand.
